packages/client/src/store/themeActions.ts

```diff
@@ -0,0 +1,97 @@
+import { createAsyncThunk } from '@reduxjs/toolkit'
+import { AxiosError, isAxiosError } from 'axios'
+import { axiosErrorHandler } from '../features/utils/axiosErrorHandler'
+import ThemeAPI from '../api/theme'
+import { Theme } from './themeSlice'
+
+interface UpdateThemeData {
+  id: number | null
+  theme: Theme
+}
+
+export const getCurrentTheme = createAsyncThunk(
+  `theme/getCurrentTheme`,
+  async (id: number, { dispatch, rejectWithValue }) => {
+    try {
+      const {
+        data: { theme },
+      } = await ThemeAPI.getCurrentTheme(id)
+
+      const localTheme = localStorage.getItem(`theme`)
```

Comment:
Тут лучше ключ темы в локал сторе вынести в константу. Вижу, что он в нескольких местах уже используется

```suggestion
      const localTheme = localStorage.getItem(THEME_LOCAL_STORE_KEY)
```